packages/ods-react/src/components/combobox/src/controller/combobox.ts

```diff
@@ -0,0 +1,109 @@
+import type { ComboboxItemOrGroup } from '../components/combobox-item/ComboboxItem';
+import type { ReactNode } from 'react';
+import { useMemo } from 'react';
+import { getElementText } from '../../../../utils/element';
+
+interface UseComboboxControllerProps {
```

Comment:
why did you use a custom syntax instead of just exposing pure functions like everywhere else on the lib?